code/index.html

```diff
@@ -3,20 +3,79 @@
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="stylesheet" href="./style.css" />  
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/745bd96b12.js" crossorigin="anonymous"></script>
   </head>
   <body>
     <header>
+      <div class="logo-container">
+        <img class="logo" src="./lakelogo_white.svg" alt="Fjord Logo" />
+        <h1>A fjord with no name</h1>
+      </div>
+      <a class="nav-link" id="about" href="#about-footer">About</a>
+      <a class="nav-link" id="blog" href="#blog-articles">Blog</a>
+      <a class="nav-link" href="#">Outdoor</a>
+      <a class="nav-link" id="contact" href="#contact-section">Contact</a>
       <!-- Some kind of logo and navbar -->
     </header>
```

Comment:
Would it be better to put all links in the nav tag? So you have a nav section in the header. In these case you dont have to give your links a class, you can just use nav tag by giving it a class name. Example: 
```
        <nav class="navbar">
            <li><a href="#">About</a></li>
            <li><a href="#">Blog</a></li>
            <li><a href="#">Outdoor</a></li>
            <li><a href="#">Contact</a></li>
        </nav>

```
I created a solution for you in attached file here. These solution shows you how I would create header section. I did media queries in tablet and mobile screens as well, so the header is responsive in many screens.  I used media queries in some other way than you. So take a look and tell me what you think. Give me a call if something is unclear :) 
[Arkiv.zip](https://github.com/Technigo/project-news-site/files/7083075/Arkiv.zip)

Reply:
I was fighting a lot with this one until I found a solution that worked for me and I will try to explain, why I choose to do it that way.
I agree that it would be much more cleaner to have the links wrapped by a navbar. But with the `<nav>` I didn't manage to change their order afterwards. 
So on Desktop they come in "normal order" like this:
![navbar_desktop](https://user-images.githubusercontent.com/77005046/131496189-ec36dcce-50cf-4b79-9727-cdd68e3daa93.png)

But on tablet I split the links up and include the logo all in one line. Two links to the left, then logo, then two links to the right.
![navbar_tablet](https://user-images.githubusercontent.com/77005046/131496208-a33f1360-6c61-49c7-ae92-56d6be894d49.png)

In your solution on the other hand the links are underneath the logo and not in the same line
![navbar_darya](https://user-images.githubusercontent.com/77005046/131497485-b2526751-1191-4a77-995f-126bb422bfc5.png)

Can you think of a more elegant version to split up the links and the logo, but stay on the same line? :)

Reply:
I understand. I found [ these](https://stackoverflow.com/questions/42442753/how-to-center-logo-between-navigation-list-items-with-flexbox) solution on Stackoverflow . In that solution they use span tag to separate links from logo. Try it!